README.md

```diff
@@ -507,6 +509,526 @@ source scripts/makeAllPlots.sh figures/ResponseUniformityMaps
+	Canv_DrawOpt = 'APE1';
+	Canv_Grid_XY = 'false,false';
+	Canv_Latex_Line = '0.62,0.86, #splitline{LS2}{Detector~Production}';
+	Canv_Latex_Line = '0.62,0.27, Gas~=~CO_{2}';
+	Canv_Legend_Dim_X = '0.20,0.60';
+	Canv_Legend_Dim_Y = '0.56,0.92';
+	Canv_Legend_Draw = 'true';
+	Canv_Log_XY = 'false,false';
+	Canv_Logo_Pos = '0';
+	Canv_Logo_Prelim = 'true';
+	Canv_Margin_Top = '0.08';
+	Canv_Margin_Bot = '0.14';
+	Canv_Margin_Lf = '0.16';
+	Canv_Margin_Rt = '0.06';
+	Canv_Name = 'LS2_Detectors';
+	Canv_Plot_Type = 'TGraphErrors';
+	Canv_Range_X = '0,3600';
+	Canv_Range_Y = '0,35';
+	Canv_Title_Offset_X = '1.0';
+	Canv_Title_Offset_Y = '1.0';
+	Canv_Title_X = 'Time #left(s#right)';
+	Canv_Title_Y = 'Pressure #left(mbar#right)';
+		[BEGIN_PLOT]
+			Plot_Color = 'kRed';
+			Plot_LegEntry = 'GE11-X-L-CERN-0002';
+			Plot_Line_Size = '1';
+			Plot_Line_Style = '1';
+			Plot_Marker_Size = '0.8';
+			Plot_Marker_Style = '20';
+			Plot_Name = 'GE11-X-L-CERN-0002';
+			[BEGIN_DATA]
+			VAR_INDEP,VAR_DEP,VAR_INDEP_ERR,VAR_DEP_ERR
+			1.0,20.38,0.0,0.0
+			62.0,20.26,0.0,0.0
+			123.0,20.13,0.0,0.0
+			184.0,20.14,0.0,0.0
+			245.0,20.11,0.0,0.0
+			306.0,20.11,0.0,0.0
+			367.0,20.04,0.0,0.0
+			428.0,20.05,0.0,0.0
+			489.0,20.03,0.0,0.0
+			550.0,20.07,0.0,0.0
+			611.0,20.05,0.0,0.0
+			672.0,20.07,0.0,0.0
+			733.0,20.01,0.0,0.0
+			794.0,20.07,0.0,0.0
+			855.0,19.98,0.0,0.0
+			916.0,19.98,0.0,0.0
+			...................
+			...................
+			...................
+			3112.0,19.39,0.0,0.0
+			3173.0,19.32,0.0,0.0
+			3234.0,19.32,0.0,0.0
+			3295.0,19.35,0.0,0.0
+			3356.0,19.23,0.0,0.0
+			3417.0,19.25,0.0,0.0
+			3478.0,19.25,0.0,0.0
+			3539.0,19.29,0.0,0.0
+			3600.0,19.2,0.0,0.0
+			[END_DATA]
+			[BEGIN_FIT]
+			Fit_Color = 'kRed';
+			Fit_Formula = '[0]*TMath::Exp([1]*x)';
+			Fit_LegEntry = 'Fit_GE11-X-L-CERN-0002';
+			Fit_Line_Size = '1';
+			Fit_Line_Style = '1';
+			Fit_Name = 'Fit_GE11-X-L-CERN-0002';
+			Fit_Option = 'R';
+			Fit_Param_IGuess = 'AMPLITUDE,6.2e-05';
+			Fit_Perform = 'true';
+			Fit_Range = '0,3600';
+			[END_FIT]
+		[END_PLOT]
+	[END_CANVAS] 
+	
+In the case that more than one input files are selected the output name is automatically set to be: **config\_QC3\_LS2\_Pres\_vs\_Time\_AllDet.cfg**
+
+**QC4_HV.py :**
+
+	--CanvTitleX=Divider Current #left(#muA#right)
+	--CanvTitleY=Applied Voltage #left(kV#right)
+	--SelectColumnX=3
+	--SelectColumnY=1
+	--SelectRowStart=3
+	--SelectRowEnd=37
+	--CanvRangeX=0,1000
+	--CanvRangeY=0,7
+	--YaxisScale
+	--LatexLines=0.62,0.86, #splitline{LS2}{Detector~Production}
+	--LatexLines=0.62,0.27, Gas~=~CO_{2}
+
+	Moreover if the Fit option is used (example: python python/QC4_HV.py —-file=QC4_GE11-X-L-CERN-0001_20170601.xlsm —-Fit)
+	the following options are set for the fit:
+
+	--Fit
+	--FitFormula=[0]*x+[1]
+	--FitParamIGuess=Req,5   (where Req is the measured resistance from the excel file) 
+	--FitRange=0,1000
+	
+Example of use: As in steps 1 and 2 of QC3.py script execute:
```

Comment:
same comment as above...restructure to be an easier copy/paste.

First give the block of code and then describe it.

Reply:
done